### ISSUE TITLE
Refresh token api call is returning undefined properties and tokens

### ISSUE THREAD

**Issue by mypark** [auth0/react-native-lock#128]
I noticed that my refreshToken calls don't error but instead return undefined for the properties:

 lock.authenticationAPI()
          .refreshToken(refreshToken)
          .then(async (authResult) => {
            //authResult has null token results
           })

Do we need to upgrade to lock 2.0?

**Comment by mypark** [comment]
Making the api request with postman - it returns a 400 and timeout error 
`"Script execution did not complete within 20 seconds. Are you calling the callback function?"`

**Comment by bz123** [comment]
I am assuming you are trying to get it from AsyncStorage and then pass it to the function above. That returns an error as you pass an undefined value by the time the function is called. Try to call the refreshToken function as a promise after you receive the refreshToken itself.

Do something like: 
`AsyncStorage.getItem('refreshToken')
    .then((refreshToken) => {
        Lock.authenticationAPI().refreshToken(refreshToken).then(async (authResult) => {console.log('newToken', authResult)});
    })`

**Comment by mypark** [comment]
No- the token is instantiated as a non null string by the time it gets there.  I also got the same error on postman and curl calling the API directly from http
